### DISCUSSION TITLE
Cirrus Images using `image_family` instead?

### DISCUSSION THREAD

**Discussion by arif-ali** [sosreport/sos#3856] [Q&A] [answered]
Is there any reason why we stick to specific images rather than go with family name.

As part of fixing obfuscation issues in our recent upload to Ubuntu we also found the same issue with `msr` load issue which avocado has picked up. Now, if we always had the latest images in Cirrus, we would have picked this up.

I already did something similar for both `report_stageone_daily_task` and `report_stagetwo_daily_task` where I am using the `image_family` instead of `image_name`, and this always picks up the latest image.

If we did change to `image_family`, we won't need to change the `cirrus.yml` on a regular basis. The only time I would change, is when we have a new release which is every 6 months .

Thoughts?

**Comment by jcastill** [comment]
I haven't found any downside to using `image_family` so far, so (after some testing) I think it should be fine from RH side.
@pmoravec any opinion on this?

**Comment by arif-ali** [comment]
in fact, just realised we don't do stagetwo tests here in CI for older releases (focal & jammy), only the latest releases. So, we wouldn't have picked this up here either.

I know this could incur extra costs and such, how would we feel about enabling stagetwo for all supported platforms?

**Reply by pmoravec** [reply]
> in fact, just realised we don't do stagetwo tests here in CI for older releases (focal & jammy), only the latest releases. So, we wouldn't have picked this up here either.
> 
> I know this could incur extra costs and such, how would we feel about enabling stagetwo for all supported platforms?

The difference is 9 releases for stageone tests vs. 6 releases for stagetwo tests. Probably not a big deal but let us check it internally.

In general, referring to image family and test always latest (plus latest minus one) seems a good "automation" approach.

**Comment by jcastill** [comment]
We'll probably need to go check internally for this before we make any changes

**Comment by TurboTurtle** [comment, marked as answer]
It's a ways in the past, but I don't believe `image_family` was available when we first took up Cirrus, so that's likely the only reason.

Fully onboard.
